lib/smart_proxy_reports/friendly_message.rb

```diff
@@ -27,6 +32,21 @@ def generate_message
     msg
   end
 
+  def assemble_message
+    "Concatenation of #{@result_tree["src"]} into #{@result_tree["dest"]}"
+  end
+
+  def dnf_message
+    package_message
+  end
+
+  def git_message
+    if @module_args_tree["clone"]
+      return "Repository #{@module_args_tree["repo"]} cloned into #{@module_args_tree["dest"]}"
+    end
+    "Check if repository #{@module_args_tree["repo"]} exists"
```

Comment:
I guess if `clone` set to `false` we could simply say something like `"Repository #{@module_args_tree["repo"]} was not cloned into #{@module_args_tree["dest"]} due to the parameter"`.

Reply:
Yeah perhaps just Repository XYZ NOT clones into XYZ.

I do not understand why this param would be useful, it is there for some reason. I guess it is pretty rare.